.github/workflows/build.yml

```diff
@@ -4,12 +4,12 @@ jobs:
   build:
     strategy:
       matrix:
-        java: [8-jdk, 11-jdk, 16-jdk, 17-jdk]
-    runs-on: ubuntu-20.04
+        java: [11-jdk, 16-jdk, 17-jdk, 21-jdk]
```

Comment:
Why Java 16? For MC 1.17? Doesn't the official MC launcher use Java 17 for that version now too?

Reply:
We can prob remove this, but as far as I'm aware it still uses Java 16

Reply:
Removed